neo/Ledger/MemoryPool.cs

```diff
@@ -434,10 +432,12 @@ internal void UpdatePoolForBlockPersisted(Block block, Snapshot snapshot)
             List<PoolItem> invalidItems = new List<PoolItem>();
             foreach (PoolItem item in unverifiedSortedTxPool.Reverse().Take(count))
             {
-                // Re-verify the top fee max high priority transactions that can be verified in a block
+                // Re-verify up to `count` transactions that can be verified in a block
+                // for high priority txs cut-off is limited to _maxTxPerBlock, while
```

Comment:
I don't think we need to specify what the cutoff is in a comment here; this code isn't related to the cutoff value.

Reply:
updated.